determined-ai/determined
# feat: introduce batch actions in user management page

## Description
[WEB-1713]




## Test Plan
- Go to User Management Table 
  - When RBAC is **disabled**
    - Check if user cannot select oneself in the checkbox
    - Check if `Action` button shows up when and only when 1 or more users are selected
    - Check if only admin users can change the status of others
    - Check if user status will be reflected as soon as applying the batch action and selection will be cleared
  - When RBAC is **enabled**
    - Check if user cannot select oneself in the checkbox
    - Check if `Action` button shows up when and only when 1 or more users are selected
    - Check if only admin users can change the status of others, user with `canModifyUsers` can `Add to Groups`, and user with `canModifyPermissions` can `Set Roles`
    - Check if user status will be reflected as soon as applying the batch action and selection will be cleared




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
